package.json

```diff
@@ -38,24 +38,24 @@
   "dependencies": {
     "co": "^4.6.0",
     "js-combinatorics": "^0.5.2",
-    "lodash": "^4.16.6",
-    "mz": "^2.5.0",
-    "request-promise": "^4.1.1",
+    "lodash": "^4.17.4",
+    "mz": "^2.6.0",
+    "request-promise": "^4.2.0",
     "require-dir": "^0.3.1",
-    "uuid": "^2.0.3",
-    "winston": "^2.3.0"
+    "uuid": "^3.0.1",
+    "winston": "^2.3.1"
   },
   "devDependencies": {
     "chai": "^3.5.0",
```

Comment:
Out current policy is that there should be no carets `^` in the `devDependencies` section. They should only exist in the `dependencies` section.